source/digits_hits/src/GateEnergySpectrumActorMessenger.cc

```diff
@@ -202,6 +202,12 @@ void GateEnergySpectrumActorMessenger::BuildCommands(G4String base)
   pEnableEnergyPerUnitMassCMD = new G4UIcmdWithABool(bb, this);
   guidance = G4String("Set energy per nucleus");
   pEnableEnergyPerUnitMassCMD->SetGuidance(guidance);
+  
+  
+  bb = base+"/setRelativeToNbPrimaryEvents";
+  pEnableRelativePrimEventsCMD = new G4UIcmdWithABool(bb, this);
+  guidance = G4String("Set values relative to number of primary events");
```

Comment:
I think you can provide a better description. Something like "Normalize all enabled 1D histograms to the number of primary events."

Reply:
True. Changed both, description and flag.